bigquery2/bigquery_endpoint.bal

```diff
@@ -87,13 +87,18 @@ public type Client client object {
 
     #  Streams data into BigQuery one record at a time without needing to run a load job.
     #
-    # + projectId - Project ID of the new table.
-    # + datasetId - Dataset ID of the new table.
-    # + rows - The rows to insert.vious call, to request the next page of results.
+    # + projectId - Project ID of the destination table.
+    # + datasetId - Dataset ID of the destination table.
+    # + tableId - Table ID of the destination table.
+    # + templateSuffix - If specified, treats the destination table as a base template, and inserts the rows into an
+    #                    instance table named "{destination}{templateSuffix}". BigQuery will manage creation of the
+    #                    instance table, using the schema of the base template table.
+    # + rows - The rows to insert.
     #
     # + return - `InsertTableData` object on success and error on failure
     public remote function insertAllTableData(string projectId, string datasetId, string tableId,
-                                              InsertRequestData[] rows) returns InsertTableData|error ;
+                                              string? templateSuffix = (), InsertRequestData[] rows)
```

Comment:
Shall we swap `templateSuffix` and `rows`, just to have an order with the required params first and then the defaultable ones? Same for L331 too.
```suggestion
                                              InsertRequestData[] rows, string? templateSuffix = ())
```